config.go

```diff
@@ -90,6 +90,13 @@ type Config struct {
 	// If Pipeline is empty, no ingest pipeline will be specified in the Bulk request.
 	Pipeline string
 
+	// RequireDataStream, If set to true, an index will be created only if a
```

Comment:
A true nitpick but could be beneficial to explain the default behavior and why one may want enable this option, to help readers unfamiliar with it.

Reply:
Documented the default behavior commit below.